CHANGELOG.md

```diff
@@ -3,11 +3,16 @@
 **Highlights**
 - Support updating workflow instance business key.
 - Support for searching workflow instances by state variable key and value.
+- Control retryable exception handling via `WorkflowSettings` (replaces deprecated `WorkflowState.isRetryAllowed(...)`)
```

Comment:
Should this be "Control exception retrying and logging via ..."?

Reply:
Yes, thanks!